config/samples/canary.yaml

```diff
@@ -84,10 +84,16 @@ spec:
           app: appinfo
         type: LoadBalancer
   tasks:
-    deployment:
-      resources:
-      - service.yaml
-      - deployment.yaml
+    - name: deployment
```

Comment:
maybe we can have just single task here to reduce verbosity

Reply:
:+1: